ethereum-consortium/docs/setupWalkthrough.md

```diff
@@ -87,11 +87,11 @@
       resource and copy the IP.  
 
       Navigate to http://{copied ip}:3001/enodes. Save the returned value
```

Comment:
This should be updated as well to staticenodes as the format for bootnodes and static enodes is different